bokehjs/test/document.coffee

```diff
@@ -553,6 +553,10 @@ describe "Document", ->
     out = stdoutTrap -> Document.from_json_string(JSON.stringify(parsed))
     expect(out).to.be.equal ""
 
+    parsed['version'] = "#{js_version}rc123"
+    out = stdoutTrap -> Document.from_json_string(JSON.stringify(parsed))
+    expect(out).to.be.equal ""
```

Comment:
I don't understand this test, or why it is passing. This is explicitly setting the python and js versions to be different, supposedly on an RC, so that *should* trigger a warning.